ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/requestDetails.test.ts

```diff
@@ -0,0 +1,188 @@
+    verifyExceptionConfirmationDetails,
+    verifySelectedCvesInModal,
+    visitWorkloadCveOverview,
+} from '../workloadCves/WorkloadCves.helpers';
+import { selectors as workloadCVESelectors } from '../workloadCves/WorkloadCves.selectors';
+import { selectors } from './ExceptionManagement.selectors';
+import { selectors as vulnSelectors } from '../vulnerabilities.selectors';
+import { visit } from '../../../helpers/visit';
+
+const deferralComment = 'Defer me';
+const deferralExpiry = 'When all CVEs are fixable';
+const deferralScope = 'All images';
+
+describe('Exception Management Request Details Page', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') ||
+            !hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    beforeEach(() => {
+        visitWorkloadCveOverview();
+        cy.get(vulnSelectors.clearFiltersButton).click(); // Note: This is a workaround to prevent a lack of CVE data from causing the test to fail in CI
+
+        // defer a single cve
+        selectSingleCveForException('DEFERRAL').then((cveName) => {
+            verifySelectedCvesInModal([cveName]);
+            fillAndSubmitExceptionForm({
+                comment: deferralComment,
+                expiryLabel: deferralExpiry,
+            });
+            verifyExceptionConfirmationDetails({
+                expectedAction: 'Deferral',
+                cves: [cveName],
+                scope: deferralScope,
+                expiry: deferralExpiry,
+            });
+            cy.get(workloadCVESelectors.copyToClipboardButton).click();
+            cy.get(workloadCVESelectors.copyToClipboardTooltipText).contains('Copied');
+            // @TODO: Can make this into a custom cypress command (ie. getClipboardText)
+            cy.window()
+                .then((win) => {
+                    return win.navigator.clipboard.readText();
+                })
+                .then((url) => {
+                    visit(url);
+                });
+        });
+    });
+
+    after(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    it('should be able to sort on the "CVE" column', () => {
+        cy.get(selectors.tableSortColumn('CVE')).should('have.attr', 'aria-sort', 'none');
```

Comment:
It might be cool in a follow up to make this a custom Cypress command as well.
i.e. 
```ts
cy.getSortedColumn('CVE', 'none');
cy.getSortedColumn('CVE', 'desc');
cy.getSortedColumn('CVE', 'asc');
```